src/windows/stumpless.def

```diff
@@ -158,4 +158,6 @@ EXPORTS
   stumpless_close_function_target               @143
   stumpless_open_function_target                @144
   stumpless_element_to_string                   @145
+  stumpless_get_cons_stream						@146
+  stumpless_set_cons_stream						@147
```

Comment:
use spaces in this file instead of tabs - that is why they are not aligned in the github rendering above